opentracing-api/src/main/java/io/opentracing/Logs.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The "Logs" class/container standardizes and simplifies common @link{io.opentracing.Span#log(Map)} patterns.
+ *
+ * @see io.opentracing.Span#log(Map)
+ */
+public final class Logs {
+    /**
+     * EVENT_KEY identifies a String log value that serves as a stable identifier for an event in a Span's lifecycle.
+     */
+    public static final String EVENT_KEY = "event";
+
+    /**
+     * MESSAGE_KEY identifies a String log value that's human-readable and descriptive. Messages need not be stable.
+     */
+    public static final String MESSAGE_KEY = "message";
+
+    /**
+     * @param eventName the value for the "event" key in the log map
+     * @return a Map suitable for use with @link{io.opentracing.Span#log(Map)}
+     * @see Logs#EVENT_KEY
+     */
+    public static Map<String, ?> event(String eventName) {
```

Comment:
FYI singleton maps are in all recent versions of java `Collections.singletonMap()`

so if we have a stable event or message key defined, then you could just as well suggest
`Collections.singletonMap(LogKeys.EVENT, "mary had a little lamb")`.

That is unless the goal is to eventually extend map with log view types, ex. 
```
class Event extends MapAdapter<String, String> {
  ...
}
```
In that case, you might want to direct them to a file like this.

I did this in Denominator, mainly to allow one to use rare or undefined DNS rdata types. It works, mostly, but has some limitations. https://github.com/Netflix/denominator/blob/master/model/src/main/java/denominator/model/rdata/SRVData.java#L23
https://github.com/Netflix/denominator/blob/master/model/src/main/java/denominator/model/ResourceRecordSets.java#L559